### PR TITLE
improvement: S3C-2289 upgrade node.js to 10.x

### PR DESCRIPTION
**Pull request by Katherinelaue1**
**What does this PR do, and why do we need it?**
Update node to 10 in development/7.4

**Which issue does this PR fix?**
fixes #S3C-2289

**Special notes for your reviewers:**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "bucketclient",
   "engines": {
-    "node": ">=6.9.5"
+    "node": ">=10.16.0"
   },
-  "version": "7.2.0",
+  "version": "7.2.1",
```
Comment (JianqinWang) [inline package.json]: shouldn't this be 7.4.x

Comment (JianqinWang) [reply package.json]: @rahulreddy

**Review by tmacro — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rahulreddy) [reply package.json]: Yes, good catch!
@Katherinelaue1 Can you please change this to `7.4.5`

**Review by JianqinWang — APPROVED** [review]
thanks!

File: package.json (hunk 1 above)
Comment (Katherinelaue1) [reply package.json]: Changed it, thanks @JianqinWang!

**Comment by rahulreddy** [comment]
@Katherinelaue1 Thanks! Please squash the commits so that we can merge.

**Comment by Katherinelaue1** [comment]
@bert-e status

**Comment by Katherinelaue1** [comment]
@bert-e approve

**Review by rahulreddy — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e bypass_leader_approval

**Pull request merged**